src/components/MultiStepForm/PersonalInformation/PersonalInformation.tsx

```diff
@@ -0,0 +1,35 @@
+import { UseFormRegister, FieldErrors } from 'react-hook-form';
+
+import { IFormInput } from '../IFormInput';
+import { FormLabel, FormInput, ErrorsLabel, ErrorContainer } from './PersonalInformationStyles';
+
+interface PersonalInformationProps {
+  register: UseFormRegister<IFormInput>;
+  errors: FieldErrors<IFormInput>;
+}
+
+const PersonalInformation: React.FC<PersonalInformationProps> = ({ register, errors }) => {
+  return (
+    <>
+      <ErrorContainer>
+        <FormLabel htmlFor="name">Name</FormLabel>
+        <FormInput {...register('name')} type="text" id="name" placeholder="e.g. Maxim Black" isNotValid={!!errors.name} />
```

Comment:
It's not a good idea to use negations in vars names, as it can lead to extra confusion. Consider using `isInvalid` or ideally `hasError`